cdap-unit-test/src/test/java/co/cask/cdap/security/AuthorizationTest.java

```diff
@@ -1058,4 +1141,22 @@ public boolean apply(Privilege input) {
     Assert.assertTrue(Sets.filter(authorizer.listPrivileges(ALICE), entityFilter).isEmpty());
     Assert.assertTrue(Sets.filter(authorizer.listPrivileges(BOB), entityFilter).isEmpty());
   }
+
+  private void verifyDatasetIsEmpty(NamespaceId namespaceId, String datasetName) throws Exception {
+    DataSetManager<KeyValueTable> outTableManager = getDataset(namespaceId.toId(), datasetName);
+    KeyValueTable outputTable = outTableManager.get();
+    Assert.assertFalse(outputTable.scan(null, null).hasNext());
+  }
+
+  private void verifyProgramFailure(Map<String, String> programArgs, ProgramManager programManager)
+    throws TimeoutException, InterruptedException {
+    programManager.start(programArgs);
+    programManager.waitForFinish(5, TimeUnit.MINUTES);
```

Comment:
Nice catch. Addressed this comment and all other comments on AuthorizationTest class in this commit: https://github.com/caskdata/cdap/pull/6427/commits/e89eec2b2ce5137d41d4f9215cfd45c1723a0036